### PR TITLE
Cookie reminder

### PR DESCRIPTION
**Pull request by KamilWo**
For now at bottom of every page, later after new implementation
closing event will be stored in cookie.

### CODE REVIEW

**Comment by KrzysiekJ** [comment]
This pull request has conflict(s).

**Comment by KamilWo** [comment]
Conflicts resolved.

**Comment by KrzysiekJ** [comment]
After cookie information dismissal and page refresh, the information is displayed again. I guess that once dismissed, it should not be displayed anymore.

**Comment by KamilWo** [comment]
Thanks, I've introduced desirable changes.

File: addons/bestja_frontend_ucw/static/src/js/custom.js [hunk 1]
```diff
@@ -43,4 +43,41 @@ jQuery(window).ready(function () {
...
+
+    $('#cookie_reminder_close').click(function() {
+        $('#cookie_reminder_container').hide();
+    });
+
+
+    /* Cookie for reminder */
+    function setCookie(cookieName, cookieValue, daysToExpire) {
+        var d = new Date();
+        d.setTime(d.getTime() + (daysToExpire * 24 * 60 * 60 * 1000));
```
Comment (KrzysiekJ) [inline addons/bestja_frontend_ucw/static/src/js/custom.js]: What does that `1000` mean?

File: addons/bestja_frontend_ucw/static/src/js/custom.js [hunk 2]
```diff
@@ -43,4 +43,41 @@ jQuery(window).ready(function () {
...
+        var name = cookieName + "=";
+        var ca = document.cookie.split(';');
+        for (var i = 0; i < ca.length; i++) {
+            var c = ca[i];
+            while (c.charAt(0) == ' ')
+                c = c.substring(1);
+            if (c.indexOf(name) == 0)
+                return c.substring(name.length, c.length);
+        }
+        //console.log('Now there is no cookie by the name: ' + cookieName);
```
Comment (KrzysiekJ) [inline addons/bestja_frontend_ucw/static/src/js/custom.js]: This should be removed.

File: addons/bestja_frontend_ucw/static/src/js/custom.js [hunk 3]
```diff
@@ -43,4 +43,41 @@ jQuery(window).ready(function () {
...
+        return "";
+    }
+
+    $('#cookie_reminder_close').click(function(){
+        setCookie('accepted','yes',7);
```
Comment (KrzysiekJ) [inline addons/bestja_frontend_ucw/static/src/js/custom.js]: Name “accepted” for that cookie is… unacceptable. :) It should be more verbose, like “cookie_info_dismissed”.

File: addons/bestja_frontend_ucw/static/src/js/custom.js (hunk 1 above)
Comment (KamilWo) [inline addons/bestja_frontend_ucw/static/src/js/custom.js]: Maybe represents miliseconds?
24 h, 60 minutes, 60 seconds, 1000 miliseconds.

File: addons/bestja_frontend_ucw/static/src/js/custom.js (hunk 2 above)
Comment (KamilWo) [inline addons/bestja_frontend_ucw/static/src/js/custom.js]: Ok, removed.

File: addons/bestja_frontend_ucw/static/src/js/custom.js (hunk 3 above)
Comment (KamilWo) [inline addons/bestja_frontend_ucw/static/src/js/custom.js]: Ok, changed.

**Comment by KrzysiekJ** [comment]
Merged in a0d3d9e.

**Pull request closed without merging**